source/delegate/contracts/Delegate.sol

```diff
@@ -388,6 +388,7 @@ contract Delegate is IDelegate, Ownable {
     uint256 priceCoef,
     uint256 priceExp
   ) internal {
+    require(priceCoef > 0, "REQUIRE_NONZERO_PRICE");
```

Comment:
We don't normally put the word `REQUIRE` in the actual require messages. Maybe something like `INVALID_PRICE_COEF` ?